wallet.go

```diff
@@ -156,19 +174,53 @@ func (w *Wallet) updateNotificationLock() {
 	w.notificationLock = noopLocker{}
 }
 
+// GetCreditAccount returns the first account that can be associated
+// with the given credit.
+// If no account is found, ErrAccountNotFound is returned.
+func (w *Wallet) GetCreditAccount(c txstore.Credit) (uint32, error) {
```

Comment:
Don't use Get in the method name for getters; it's not idiomatic Go (same thing below, for GetDebitAccount).

Also, would it make sense to return both the account number and name here, or is that an expensive lookup and isn't needed in all places?